### PR TITLE
Remove fresh start player type

### LINKED ISSUE #1129: There are no longer any fresh start accounts
They have all been transitioned to regular accounts.

### PR DESCRIPTION
**Pull request by Jonxslays**
This PR removes the fresh start player type and converts existing fresh start players to regular.
FSW players who move to the main game will no longer be updated, i suppose people could reassign their player type.
If you want to keep fresh start around just go ahead and close the PR.

I got some good experience in either way :D.

I generated the migrations and updated everywhere I could find mention to fresh start.
Definitely look this over carefully though to make sure I didn't miss anything.
Bumped the server and client-js versions because I think I remember you saying that bumping client-js will pull in the new types.

Closes #1129

### CODE REVIEW

File: server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql
Comment (psikoi) [inline server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql:1]: I thought Prisma didn't allow `UPDATE` migrations 🤔 

This PR looks good, I'm just wondering if all FSW accounts have been transfered to the main game, or is there some step you need to do ingame to become a main acc.

Comment (Jonxslays) [reply server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql:1]: According to https://oldschool.runescape.wiki/w/Fresh_Start_Worlds all accounts were automatically transferred **except** accounts with ge offers in, which have to move themselves. 

As far as the update migration, I just created a blank migration and wrote the sql myself. It applied normally and did update fresh start accounts to regular on my local machine.

Comment (psikoi) [reply server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql:1]: Okay, i might run that migration on production manually actually

So if you could remove that file, PR's good to merge!

Comment (Jonxslays) [reply server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql:1]: Ok I'll remove it, the other migration gives warnings about failing if there are any fresh start types in the db so just make sure to do that update before merging this.

Comment (psikoi) [reply server/prisma/migrations/20230422030446_update_fresh_start_players_to_regular/migration.sql:1]: yep, done, thanks 🙏

**Pull request merged**